go/vt/vtctl/vtctl.go

```diff
@@ -1845,21 +1845,27 @@ func commandMigrateReads(ctx context.Context, wr *wrangler.Wrangler, subFlags *f
 	if *reverse {
 		direction = wrangler.DirectionBackward
 	}
-	return wr.MigrateReads(ctx, keyspace, workflow, servedType, cells, direction)
+	if *workflow == "" {
```

Comment:
How does this affect `vtctl SplitClone`? should that return an workflow name that can be used for subsequent invocations of `MigrateReads` and `MigrateWrites`?

Reply:
vtctl SplitClone and VerticalSplitClone will also be deprecated. They won't work with MigrateReads and MigrateWrites. The replacement commands are `Migrate` and `Reshard`, which you'll see in the POC branch.

Reply:
+1